portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,26 @@ function randomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = Fact;
 }
+function getName(){
+    
+     fetch('/data').then(response => response.json()).then((list) => {
+         console.log(list[0])
+    const json = document.getElementById('list-container');
+    json.innerHTML = '';
+    json.appendChild(
+        createListElement('1: ' + list[0]));
+    json.appendChild(
+        createListElement('2: ' + list[1]));
+    json.appendChild(
+        createListElement('3: ' + list[2]));
```

Comment:
Could use a [`for..of` loop](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Statements/for...of):
```suggestion
   for (const text of list) {
     const liElement := document.createElement('li');
     liElement.innerText = text;
     json.appendChild()
   }
```

Reply:
FYI, this is still something you could do

Reply:
A shorter alternative:
```javascript
list.map(createListElement).forEach(e => json.appendChild(e));
```